docs/interface_dtfj.md

```diff
@@ -96,8 +96,8 @@ public class DTFJEX1 {
         if (args.length > 0) {
             File f = new File(args[0]);
             try {
-                Class factoryClass = Class.forName("com.ibm.dtfj.image.j9.ImageFactory");
-                ImageFactory factory = (ImageFactory) factoryClass.newInstance();
+                Class<?> factoryClass = Class.forName("com.ibm.dtfj.image.j9.ImageFactory");
+                ImageFactory factory = (ImageFactory) factoryClass.getDeclaredConstructor().newInstance();
```

Comment:
Using this pattern means that `InvocationTargetException` and `NoSuchMethodException` (or `ReflectiveOperationException`) must also be caught and handled.

Reply:
I think generalizing `InstantiationException` to `ReflectiveOperationException` makes sense here.

Reply:
Ahh, right, thanks. I'll work on that.

Reply:
@keithc-ca Fixed, please re-check